editor/js/Editor.js

```diff
@@ -347,6 +347,38 @@ Editor.prototype = {
 
 	},
 
+	getObjectMaterial: function ( object, slot ) {
+
+		var material = object.material;
+
+		if( Array.isArray( material ) == true){
+			var slot = slot | 0;
+
+			if(slot <  0) slot = 0;
```

Comment:
It was a hedge against the UI possibly spitting out a negative number.. I can remove it.